src/writer/app_runner.py

```
@@ -430,8 +430,31 @@
         import io
         from contextlib import redirect_stdout
 
+        import writerai
```

Pylint (3.3.7)

[convention] Import outside toplevel (writerai)

(C0415)